content/posts/knowledge/Laptop Hardware Security.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Laptop Hardware Security"
+date: 2024-06-10
+tags: ['Knowledge base', 'Hardware', 'Security']
+author: Tommy
+---
+
+While browsing privacy forums, I often see a lot discussions regarding laptop hardware security, especially in regards to resistance against physical attacks. Unfortunately, most of the advice given on these forums are incorrect, recommending people to buy overpriced products that are significantly less secure than standard laptops.
+
+In this post, I will walk you through a quick overview of how hardware security is generally implemented for laptops, what to look for, and what to avoid. We will not discuss MacBooks or Chromebooks, as they are vastly different from normal x86 Windows/Linux hardware.
+
+![Latitude 9440](/images/latitude-9440.jpg)
```

Comment:
The Latitude 9440 is not a recommendable laptop model for Linux usage and should not be shown in this post. Showcase a recommendable model instead, such as the Latitude 5440.

Reply:
Come on it looks beautiful. 5440 is ugly. Worst case I will use Precision 5490 if you insist

Reply:
Okay just change it to a non descrip one from Dell's unsplash